app/src/main/java/com/chartiq/chartiqsample/studies/StudyOptionsActivity.java

```diff
@@ -261,7 +261,26 @@ public void onClick(View v) {
         if ("auto".equals(parameter.color)) {
             color.setBackgroundColor(Color.BLACK);
         } else {
-            color.setBackgroundColor(Color.parseColor(parameter.color));
+            try{
+                if(parameter.color.contains("rgb")) {
+                    String subString = parameter.color.substring(parameter.color.indexOf('(') + 1, parameter.color.indexOf(')'));
+                    String rgbColors[] = subString.split(",");
+                    int parsedColor = 0;
+                    if(rgbColors.length == 4) { // contains an alpha value
+                        parsedColor = Color.argb(Integer.parseInt(rgbColors[3].trim()), Integer.parseInt(rgbColors[0].trim()),
```

Comment:
Can you simplify this a bit?  Like if there are only 3 arguments, just set the alpha to 1, and then call Color.argb regardless of whether there are 3 or 4 arguments.

Reply:
Sure, that's fine.